test/integration/integration_admin_test.cc

```diff
@@ -53,6 +53,55 @@ TEST_P(IntegrationAdminTest, HealthCheck) {
   EXPECT_EQ("200", response->headers().Status()->value().getStringView());
 }
 
+TEST_P(IntegrationAdminTest, HealthCheckWithoutServerStats) {
+  envoy::config::metrics::v2::StatsMatcher stats_matcher;
+  stats_matcher.mutable_exclusion_list()->add_patterns()->set_prefix("server.");
+  initialize(stats_matcher);
+
+  BufferingStreamDecoderPtr response = IntegrationUtil::makeSingleRequest(
+      lookupPort("http"), "POST", "/healthcheck", "", downstreamProtocol(), version_);
+  EXPECT_TRUE(response->complete());
```

Comment:
Consider factoring out these 3 lines into a local lambda `request` factoring out the `EXPECT_TRUE(response->complete())` and then you can write:

  EXPECT_EQ("200", request("http", "POST", "/healthcheck", response));

Then you can write these testcases more concisely and it will be easier to read because there'll be less boilerplate.

Reply:
Addressed in e86dae0.